public/js/auth/login.js

```diff
@@ -41,8 +41,10 @@ document.addEventListener('DOMContentLoaded', function () {
         const btnEnviarCodigo = document.getElementById('btn_enviar_codigo');
         btnEnviarCodigo.classList.add('disabled');
         const codigo = getFullCode();
+        const rut = rutInput.value;
         resetMessage(errorCodigo, successCodigo);
         let data = {
+            rut: rut,
```

Comment:
crearte una variable rut pero solo se utiliza una vez no es mejor dejarlo directo ? 

````suggestion
rut: rutInput.value,

Reply:
oki doki